pkg/apis/core/v1/defaults.go

```diff
@@ -23,6 +23,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/kubernetes/pkg/util/parsers"
+	"k8s.io/utils/pointer"
```

Comment:
this is already imported one line below :)

Although, if there is no conflicts, it's better to to set an alias. In other words, you can keep line 26, but remove 27.